controllers/contacts/index.js

```diff
@@ -0,0 +1,15 @@
+const getListContact = require("./getListContact");
+const getContact = require("./getContact");
+const addContacts = require("./addContacts");
+const removeContact = require("./removeContact");
+const updateContact = require("./updateContact");
+const updateStatus = require("./updateStatus");
+
+module.exports = {
```

Comment:
all those controllers you could just store in one file called contact.controller.js. No need to separate  them for different files